device/router.yaml

```diff
@@ -0,0 +1,20 @@
+components:
+  schemas:
+    Router:
+      x-include: 
+      - '../common/common.yaml#/components/schemas/Named.Object'
+      description: >-
+        This is emulating a single router running one or more protocols, 
```

Comment:
This is emulating a single router running --> This emulates a router running one or more protocols like BGP, ISIS, OSPFv2, RSVP, etc.